### PR TITLE
Fix content packs- Reut

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

relates: https://github.com/demisto/etc/issues/23712

Packs that were fixed:
* CommonScripts
* NIST
* Malware- ignored IF107 errors
* ML - ignore SC100 error

### CODE REVIEW

File: Packs/CommonScripts/Scripts/NumberOfPhishingAttemptPerUser/CHAGELOG.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-## [Unreleased]
-- Added NumberOfPhishingAttemptPerUser automation. Tagged as dynamic-section. Shows a bar chart of the number of incident the 'To' and 'From' email addresses
-  appeared on.
```
Comment (ronykoz) [inline Packs/CommonScripts/Scripts/NumberOfPhishingAttemptPerUser/CHAGELOG.md:3]: why did you remove this

Comment (reutshal) [reply Packs/CommonScripts/Scripts/NumberOfPhishingAttemptPerUser/CHAGELOG.md:3]: Because the name was missing an N and there is already a new changelog
